packages/material-ui/src/Slide/Slide.js

```diff
@@ -35,23 +36,41 @@ function getTranslateValue(direction, node) {
   }
 
   if (direction === 'left') {
+    if (targetNode) {
+      return `translateX(${rect.left + targetNode.offsetWidth}px) translateX(${
```

Comment:
Something is off when using the `direction="left"`. See https://codesandbox.io/s/wu05t?file=/demo.js

Reply:
Thanks for report this issue. I will fix it.

Reply:
This issue is fixed.